contrib/release/build.sh

```diff
@@ -257,7 +353,7 @@ linux*)
 
 	# fbc sources
 	cp -R ../input/fbc fbc
-	cd fbc && git reset --hard "$fbccommit" && cd ..
+	cd fbc && git reset --hard "$remote_name"/"$fbccommit" && cd ..
```

Comment:
If `$fbccommit` is a SHA1 hash then it will break, I think something like `origin/7fbad71f6508342a57acbf7527f456c794bae2b0` doesn't work.